### PR TITLE
Add adobe-cloud-manager permissions YAML file

### PR DESCRIPTION
**Pull request by ahmed-musallam**
Per: https://issues.jenkins-ci.org/browse/HOSTING-968

Adding adobe-cloud-manager permissions YAML file in this PR.

The sole maintainer/updater is currently me: @ahmed-musallam

# Description

https://github.com/jenkinsci/adobe-cloud-manager-plugin

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [x] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [x] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [x] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by ahmed-musallam** [comment]
Please let me know if I need to provide anything else

File: permissions/plugin-adobe-cloud-manage [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "adobe-cloud-manager"
+github: "jenkinsci/adobe-cloud-manager-plugin"
+paths:
+- "org/jenkins-ci/plugins/adobe-cloud-manager"
```
Comment (slide) [inline permissions/plugin-adobe-cloud-manage:5]:
```suggestion
- "io/jenkins/plugins/adobe-cloud-manager"
```

See https://github.com/jenkinsci/adobe-cloud-manager-plugin/blob/master/pom.xml#L10

Comment (ahmed-musallam) [reply permissions/plugin-adobe-cloud-manage:5]: Ahh! great catch! thank you!

**Comment by slide** [comment]
Just one comment, if you are working for Adobe, it's also good to add a "company account" to the release list. This is good so that if you move on at some point, there is still an account that has access to release the plugin.

**Comment by ahmed-musallam** [comment]
@slide, I do not work for Adobe, But I'll reach out to someone there to see if they'd also like to be added as an admin or contributor

**Comment by ahmed-musallam** [comment]
@slide the pr-merge build failed because the filename was missing the ending `r`, I have changed the file name as you can see in my last commit bu I thing the build shows it as `No changes` and the job is done, but says `pending` on here. Am I reading this correctly ?

**Comment by ahmed-musallam** [comment]
@slide disregard previous comment, turns out my file name was missing the `.yml` extension.

Fixed it and raised another PR for a more helpful error message: #1551

**Comment by ahmed-musallam** [comment]
@slide looks like this is good to be merged. Thanks for all your help!

**Comment by slide** [comment]
You don't have to add anyone from Adobe, just wondered.

**Pull request merged**